api-java-mixin.raml

```diff
@@ -620,18 +976,14 @@ types:
           return instance;
       }
   HighPrecisionMoney:
+    (java-extends): 'HighPrecisionMoneyMixin'
     (java-mixin): |
       public static HighPrecisionMoney of(final javax.money.MonetaryAmount monetaryAmount, final int fractionDigits) {
           return MoneyUtil.of(monetaryAmount, fractionDigits);
       }
-      public default HighPrecisionMoneyDraft toDraft() {
-          return HighPrecisionMoneyDraft.of(this);
-      }
-      @Override
-      default javax.money.MonetaryOperator createMoneyOperator() {
```

Comment:
Please revert this as these methods override the createMoneyOperator from the Money class. Also it seems to work this could lead to confusion because of the override directive